.github/workflows/ci.yml

```diff
@@ -56,6 +56,7 @@ jobs:
           path: tests/.coverage*
           if-no-files-found: ignore
           retention-days: 1
+          include-hidden-files: true
```

Comment:
there's another one below (line 100something)

Reply:
Ah, missed that, thanks!